score/mw/com/impl/configuration/config_parser.cpp

```diff
@@ -548,11 +654,19 @@ auto ParseServiceElementTracingEnabled(const score::json::Any& json,
 
 auto ParsePermissionChecks(const score::json::Any& deployment_instance) -> std::string_view
 {
-    const auto permission_checks =
-        deployment_instance.As<score::json::Object>().value().get().find(kPermissionChecksKey.data());
-    if (permission_checks != deployment_instance.As<score::json::Object>().value().get().cend())
+    auto deployment_obj = deployment_instance.As<score::json::Object>();
+    if (!deployment_obj.has_value()) {
+        return kDefaultPermissionChecks;
```

Comment:
Behavioral change. Should terminate.

```suggestion
        score::mw::log::LogFatal("lola") << "Deployment instance not parsable.";
        SCORE_LANGUAGE_FUTURECPP_ASSERT_PRD(false);
```